src/common/SPELLS/others.js

```diff
@@ -216,6 +236,51 @@ export default {
     name: 'Bountiful Captain\'s Feast',
     icon: 'inv_cooking_80_majorfeast',
   },
+  FAMINE_EVALUATOR_AND_SNACK_TABLE_FEFAST_INT: {
+    id: 297117,
+    name: 'Famine Evaluator And Snack Table',
+    icon: 'trade_archaeology_staffofsorcerer_than-thaurissan',
+  },
+  FAMINE_EVALUATOR_AND_SNACK_TABLE_FEFAST_STR: {
+    id: 297118,
+    name: 'Famine Evaluator And Snack Table',
+    icon: 'trade_archaeology_staffofsorcerer_than-thaurissan',
+  },
+  FAMINE_EVALUATOR_AND_SNACK_TABLE_FEFAST_AGI: {
+    id: 297116,
+    name: 'Famine Evaluator And Snack Table',
+    icon: 'trade_archaeology_staffofsorcerer_than-thaurissan',
+  },
+  FAMINE_EVALUATOR_AND_SNACK_TABLE_FEFAST_STA: {
+    id: 297119,
+    name: 'Famine Evaluator And Snack Table',
+    icon: 'trade_archaeology_staffofsorcerer_than-thaurissan',
+  },
+  ABYSSAL_FRIED_RISSOLE: {
+    id: 297035,
+    name: 'Abyssal-Fried Rissole',
+    icon: 'inv_misc_fish_18',
+  },
+  BIL_TONG: {
+    id: 297037,
+    name: 'Bil\'Tong',
+    icon: 'inv_misc_food_48',
+  },
+  FRAGRANT_KAKAVIA: {
+    id: 297040,
+    name: 'Fragrant Kakavia',
+    icon: 'inv_misc_food_159_fish_white',
+  },
+  MECH_DOWEL_BIG_MECH: {
```

Comment:
DOWELS not DOWEL

Reply:
https://ptr.wowhead.com/item=168310/mech-dowels-big-mech